src/installer.lisp

```diff
@@ -17,29 +17,38 @@
 (defclass user-image-installer (jupyter:user-image-installer maxima-installer)
   ())
 
-(defclass system-installer (jupyter:system-installer maxima-installer)
+(defclass system-installer (jupyter:system-bundle-installer maxima-installer)
```

Comment:
Why did you change this? The system-bundle-installer is not same as the system-installer. I think this should be reverted (and tested),